src/commands/balance.ts

```diff
@@ -0,0 +1,126 @@
+import { CommandInteraction, SlashCommandBuilder, EmbedBuilder } from 'discord.js';
+import { checkUsers, checkBalance, changeBalance, addUsers, checkUserWalletsUser, addUserWalletsUser } from '../DBUtils';
+import { replyWithEmbed } from '../utils';
+
+export const Balance = {
+    info: new SlashCommandBuilder()
+        .setName('balance')
+        .setDescription('Shows you a users JeffreyCoin balance')
+        .addUserOption(option =>
+            option.setName('member')
+                .setDescription('Which member you would like to check (defaults to self)')
+                .setRequired(false))
+        .addIntegerOption(option =>
+            option.setName('add')
+                .setDescription('Add to balance, negative number to subtract. (officer+ only)')
+                .setRequired(false)),
+
+    run: async (interaction: CommandInteraction): Promise<void> => {
+        if (!interaction.guild) {
+            await interaction.reply('This command can only be done in a server.');
+            return;
+        }
+        const userID = interaction.user.id;
+        const member = interaction.options.getUser('member');
+        const getRole = interaction.guild.roles.cache;
+        const add = (interaction.options as any).getInteger('add');
```

Comment:
Try to avoid using `any` casts.

This might work:
```ts
interaction.options.getInteger('add') 
```

Make sure your `interaction` type is `ChatInputCommandInteraction`